extensions/linux/dictionary.json

```diff
@@ -17,6 +17,11 @@
       "caption": "Effective Group ID",
       "description": "The effective group under which this process is running.",
       "type": "integer_t"
+    },
+    "kernel_ver": {
```

Comment:
Should we call it `kernel_rel` for "Kernel Release" to better match the intended "uname -r" guidance?

Reply:
That is what I was originally thinking. I will update.

Reply:
Sorry about that.

Reply:
Updated to kernel_release